docs/.vuepress/navbar.js

```diff
@@ -2,7 +2,7 @@ const navbar_en = [
   {
     text: "Start Here",
     children: [
-      '/docs/user_guide/',
+      '/docs/user-guide',
```

Comment:
```suggestion
      '/docs/user_guide',
```
Not sure I understand the change here.
I don't think this links anywhere as the dir is still `docs/user_guide` so doesn't either the dir need to change or this link?

Reply:
Uh because that should've been redirected, and I didn't change it in the navbar....whoops. This is gonna be fixed in a second.